include/ica_api.h

```diff
@@ -68,10 +68,38 @@ typedef ica_adapter_handle_t ICA_ADAPTER_HANDLE;
 #define ICA_CBCCS_VARIANT2	2
 #define ICA_CBCCS_VARIANT3	3
 
+/**
+ * ICA flags
+ */
 #define ICA_FLAG_SHW 4 /* static hardware support (symmetric ops - CPACF) */
 #define ICA_FLAG_DHW 2 /* dynamic hardware support (asymmetric ops - CEX) */
 #define ICA_FLAG_SW  1 /* software implementation (fallback / backup) */
 
+/**
+ * ICA properties: key lengths
+ */
+#define ICA_PROPERTY_AES_128		0x00000001
+#define ICA_PROPERTY_AES_192		0x00000002
+#define ICA_PROPERTY_AES_256		0x00000004
+
+#define ICA_PROPERTY_RSA_512		0x00000001
+#define ICA_PROPERTY_RSA_1024		0x00000002
+#define ICA_PROPERTY_RSA_2048		0x00000004
+#define ICA_PROPERTY_RSA_4096		0x00000008
+
+#define ICA_PROPERTY_RSA_ALL	\
+	(ICA_PROPERTY_RSA_512 |		\
+	 ICA_PROPERTY_RSA_1024 |	\
+	 ICA_PROPERTY_RSA_2048 |	\
+	 ICA_PROPERTY_RSA_4096)
+
+#define ICA_PROPERTY_EC_BP			0x00000001 /* Brainpool curves */
+#define ICA_PROPERTY_EC_NIST		0x00000002 /* NIST curves */
```

Comment:
Is there a whitespace error ? The NIST curves constant is indented differently than the others in the GitHub diff view....

Reply:
I will fix this.